code/obj/item/storage/bible.dm

```diff
@@ -45,6 +46,8 @@ var/global/list/bible_contents = list()
 				if( prob(25) )
 					H.cure_disease_by_path(/datum/ailment/disability/clumsy/cluwne)
 			M.HealDamage("All", heal_amt, heal_amt)
+			if(prob(5))
```

Comment:
prob(5) != 3 minute cooldown

Reply:
Outdated note from when I wanted to use sleep originally until Kyle suggested using probs

Reply:
don't get either of these comments

Reply:
I originally wanted Job XP for certain actions to be on a 3 minute cooldown when rewarded. However Kyle pointed out that that would be taxing having to keep all those keep tracking of it all. So I did what he suggested, which was to make it based on probability. I forgot to remove a comment that basically said 180 seconds = 3 minutes so that people wouldnt have to work it out for themselves